### ISSUE TITLE
import-jdl does not report unknown data type correctly

### ISSUE THREAD

**Issue by bhelm** [jhipster/generator-jhipster#7861]
##### **Overview of the issue**
jhipster import-jdl reports a reserved keyword error but actually should report an invalid data type.

##### **Reproduce the error**
try to import this entity:
```
entity TestAuction {
    starttimess234jnsdkfnjsd DateTime required,
    endtime DateTime required,
}
```
->
```
[xra@work test]$ jhipster import-jdl jhipster-jdl3.jh
Using JHipster version installed locally in current project's node_modules
Executing jhipster:import-jdl jhipster-jdl3.jh
Options: 
The JDL is being parsed.
Error: The name 'starttimess234jnsdkfnjsd' is a reserved keyword and can not be used as an entity field name.
```
Changing DateTime to ZonedDateTime solves the issue, but the importer should state a correct error instead of a misleading one.

##### **JHipster Version(s)**

```
cartest@0.0.0 /home/xra/workspace/jhipster/test
└── generator-jhipster@5.0.1 

```

##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**

<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "promptValues": {
      "packageName": "com.hw.test",
      "nativeLanguage": "de"
    },
    "jhipsterVersion": "5.0.0",
    "applicationType": "monolith",
    "baseName": "cartest",
    "packageName": "com.hw.test",
    "packageFolder": "com/hw/test",
    "serverPort": "8080",
    "authenticationType": "jwt",
    "cacheProvider": "ehcache",
    "enableHibernateCache": true,
    "websocket": false,
    "databaseType": "sql",
    "devDatabaseType": "mysql",
    "prodDatabaseType": "mysql",
    "searchEngine": "elasticsearch",
    "messageBroker": false,
    "serviceDiscoveryType": false,
    "buildTool": "maven",
    "enableSwaggerCodegen": false,
    "jwtSecretKey": "replaced-by-jhipster-info",
    "clientFramework": "angularX",
    "useSass": false,
    "clientPackageManager": "yarn",
    "testFrameworks": [],
    "jhiPrefix": "jhi",
    "enableTranslation": true,
    "nativeLanguage": "de",
    "languages": [
      "de",
      "en",
      "pl"
    ]
  }
}
</pre>
</details>

##### **Environment and Tools**

java version "1.8.0_172"
Java(TM) SE Runtime Environment (build 1.8.0_172-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.172-b11, mixed mode)

git version 2.17.1

node: v10.5.0

npm: 6.1.0

yeoman: 2.0.2

yarn: 1.7.0

Docker version 18.05.0-ce, build f150324782

docker-compose version 1.21.2, build unknown

**Comment by deepu105** [comment]
cc @MathieuAA

**Comment by MathieuAA** [comment]
@deepu105 Either I don't pay attention to the notifications, or I'm going blind. Maybe both. I'll create an issue for this in JCore to link to this one and deal with this tomorrow. I'm going to close this.

**Comment by MathieuAA** [comment]
It's fixed and will be shipped with v3.1.0
